src/cli.rs

```diff
@@ -28,5 +28,5 @@ pub struct Cli {
   pub port: Option<u16>,
 
   #[arg(long = "database", value_name = "DATABASE", help = "Name of database for connection (ex. postgres)")]
-  pub database: Option<String>,
+  pub database: String,
```

Comment:
this is actually referring to the name of the specific database within a postgres/mysql instance that is being connected to (not sure if sqlite has something similar), for example, postgres://root:password@localhost:5432/rainfrog, rainfrog is the "database" / "db_name"

for postgres vs mysql vs sqlite, we'll need a new arg, let's call it `driver`